spec/helper.rb

```diff
@@ -1,2 +1,3 @@
 require 'rspec'
 require 'email_doc'
+require 'active_support/rescuable'
```

Comment:
Aha, but I actually feel that it is the responsibility of action_mailer...🤔  (I mean that [action_mailer/rescuable.rb](https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer/rescuable.rb#L6) should require [active_support/rescuable.rb](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/rescuable.rb) ??

Anyway, this change is OK for me, thank you 👍